apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -294,6 +296,11 @@ def prepare_tag(
             )
 
         tag, depends = get_graph_info(html_tags)
+        if SPEC_CHAR.intersection(frozenset(tag)):
```

Comment:
```suggestion
        if SPEC_CHAR.intersection(tag):
```
Conversion is not needed.